src/utils/formatting-doc-to-txt.ts

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+
+const MAX_ANSWERS = 8;
+
+const inputFilename = 'input.txt';
+const outputFilename = 'output.csv';
+
+// Read the input file and split it into lines
+const lines = fs.readFileSync(inputFilename, 'utf8').split('\n');
+
```

Comment:
This whole part can be a function as well, processLines ( up to line 73 ). It's purpose would be to process the given lines.